lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -14,8 +14,26 @@ class BulkDataPatientExportSequence < SequenceBase
       requires :token
       conformance_supports :Patient
 
-      def check_export_kick_off(klass)
-        reply = export_kick_off(klass)
+      attr_accessor :run_all_kick_off_tests
+
+      def check_export_kick_off(klass, search_params: nil)
+        @search_params = search_params
+
+        reply = export_kick_off(klass, search_params: search_params)
+
+        # Servers unable to support _type SHOULD return an error and OperationOutcome resource
+        # so clients can re-submit a request omitting the _type parameter.
+        if search_params&.key?('_type') && reply.code >= 400
+          response_body = JSON.parse(reply.body)
+          message = response_body['issue']
+            .map { |issue| issue['diagnostics'].presence }
+            .compact
+            .join(' ')
+
+          skip message
+        end
+
+        @server_support_type_parameter = search_params&.key?('_type')
```

Comment:
I think this should be named `@server_supports_type_parameter`. Also, it should be moved above line 26 so that it can be used in the condition rather than repeating `search_params&.key?('_type')`